src/main/scala/tech/cryptonomic/conseil/util/ConfigUtil.scala

```diff
@@ -0,0 +1,61 @@
+package tech.cryptonomic.conseil.util
+
+import com.typesafe.config.ConfigFactory
+import scala.collection.JavaConverters._
+
+import java.util.HashMap
+
+object ConfigUtil {
+
+  private val conf = ConfigFactory.load
+
+  /**
+    * Output Map, keys being blockchain platforms, values being comma and space
+    * separated string of their networks
+    * @return platform, network Map
+    */
+  def parsePlatformsAndNetworks(): Map[String, String] = {
+    val obj = conf.getObject("platforms").asScala.toMap
+    val platformsAndNetworks = obj.map { pair =>
```

Comment:
You can pattern match anytime you use a lambda in the `{ case .... }` form

I'll spend a few words here to make it clear:
``` { case x => doSomething(x) }``` is syntax shortcut to define a `PartialFunction[A, B]` which is a function object that is only defined on those input values where the pattern matches (it's a partial function as opposed to a total function, meaning that it's not defined on the whole domain declared, i.e. the `A` type)

You can usually pass such PF literals when a function is expected, with the caveat that you must consider what happens when the input value is not handled.

The advantage you get is that you can directly "decompose" the input values, especially tuples or case classes, to make the code less cluttered and more to the point.

In this case it would look like

```scala
  val platformsAndNetworks = obj.map {
    case (platforms, networkMap) =>
      val networks = networkMap.unwrapped().asInstanceOf...
  }
```